Delicate_Flowers_Vignette-AR.qmd

```diff
@@ -0,0 +1,222 @@
+
+
+diary_whole_unnest <- diary_whole %>% 
+  unnest_tokens(word, text)  %>% 
+ filter(str_detect(word, "[a-z']$"))  %>%
+  anti_join(stop_words_custom)# eliminates stop words 
+
+diary.dtm <- diary_whole_unnest %>% 
+  count(file_name, word) %>% 
+  cast_dtm(file_name, word, n)
+```
+
+### Reading the Texts
+
+In order to responsibly text mine this data, I had to ground myself in the text. For me, this meant reading a number of the diaries in my corpus cover to cover to get a feel for what they talked about, how they felt, etc. I was immediately struck by the brazen defiance of some of these women. Consider, for instance, Sarah Morgan Dawson. She lived in Louisiana during the war, and, when the Union army took New Orleans, she immediately established a plan for how she would combat her enemy. When they ordered that "all devices, signs, and flags of the Confederacy shall be suppressed," she responded with a resolve to make new ones. And when those were taken, she would make even more. If they tried to remove the flag she kept pinned to her chest, "the man who dares attempt it - well! a pistol in my pocket fills up the gap."[^delicate_flowers_vignette-ar-2] Dawson saw herself as an active participant in the war effort and was willing to resort to physical violence in the name of the Confederacy. On many occurrences, the war excited her.
+
+[^delicate_flowers_vignette-ar-2]: [Sarah Morgan Dawson, “A Confederate Girl’s Diary: 1842-1909,” (Documenting the South: UNC. Accessed February 12, 2024)](https://www.zotero.org/google-docs/?broken=MNVVUt), 24.
+
+### Word Vector Analysis
+
+Word vector analysis seemed like the obvious choice to distantly view this corpus. Each of these women had a unique writing style and their own preferred vocabularies. Additionally, Southern white women were accustomed to using euphemisms to discuss delicate topics, such as sex.[^delicate_flowers_vignette-ar-3] Word vectors promised to mitigate these complications.
+
+[^delicate_flowers_vignette-ar-3]: E. Susan Barber and Charles F. Ritter, “Dangerous Liaisons: Working Women and Sexual Justice in the American Civil War.” European journal of American studies 10, no. 1 (March 26, 2015).
+
+```{r}
+#| eval: false
+# Need to wrap these in an if statement so that if the file generated exists, it doesn't rerun the model, it just loads the model file (.txt or .bin respectively)
+prep_word2vec(origin="text files",destination="textfiles.txt",lowercase=T,bundle_ngrams=1)
+
+model = train_word2vec("textfiles.txt","textfiles.bin",vectors=400,threads=6,window=12,iter=3,negative_samples=5)
```

Comment:
We discussed this in that previous pull request but for some reason its not in the final code. Make note of this for when you work on this again.